base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/blocks/BlockDetailsViewModel.java

```diff
@@ -123,6 +123,10 @@ public void updateBlock() {
     	editingBlock.setIsVisible(visible);
     }
     
+    public void validate() {
```

Comment:
Would it be possible to add the validate to the constructor instead? Then the validation code can remain private and it will be validated on construction. Without having to call a method?
Also can you add some java doc to this method if you keep it please?